app/uk/gov/hmrc/cgtpropertydisposalsfrontend/services/returns/ReturnsService.scala

```diff
@@ -611,7 +611,7 @@ class ReturnsServiceImpl @Inject() (
   ): EitherT[Future, Error, List[ReturnSummary]] = {
     val today    = LocalDate.now()
     val fromDate = today.minusYears(viewConfig.numberOfTaxYearsForReturns).plusDays(1L)
-    val toDate   = today
+    val toDate   = TimeUtils.getTaxYearEndDateInclusive(today)
```

Comment:
I'd put a comment here (or in the connector) explaining why we're supplying toDate as the end of the tax year (and that it wouldn't work with LocalDate.now()).

Reply:
done